index.html

```diff
@@ -332,130 +323,134 @@
       </h2>
       <p>
         The terms <dfn><a href=
```

Comment:
OK, I had hesitated as I did not want that section to grow too much in size. Done in latest commit.